src/applications/appeals/shared/components/FileField.jsx

```diff
@@ -669,6 +686,7 @@ FileField.propTypes = {
   formContext: PropTypes.shape({
     onReviewPage: PropTypes.bool,
     reviewMode: PropTypes.bool,
+    submitted: PropTypes.bool,
```

Comment:
All the changes in this file are also in the [Supplemental  PR](https://github.com/department-of-veterans-affairs/vets-website/pull/32677) - included to keep each PR independent